jsdoc-template/static/styles/jsdoc.css

```diff
@@ -59,6 +59,7 @@ h1 {
   font-weight: 300;
   font-size: 48px;
   margin: 1em 0 .5em;
+  line-height: normal;
```

Comment:
I know this is an old PR, but I was just about to make one to fix the same bug I noticed. I would recommend changing this to `line-height: 1.6`, though. I think it would preserve the original intention of the `line-height: 160%` better, because [line-height percentages don't work like we want them to with inheritance](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)